WordPress/gutenberg
# Update HeightControl component to label inputs

## What?
Adds accessible names to both inputs in the `HeightControl` component, replacing the `fieldset` wrapper and using `aria-controls` instead.

## Why?
Currently, `HeightControl` uses a `<fieldset>` to label the input fields, but this doesn't provide an accessible name for either. [Both fields should have an appropriate name](https://www.w3.org/WAI/WCAG21/Understanding/labels-or-instructions).

## How?
* Updates the `RangeControl` component to accept an optional ID.
* Replaces the `<fieldset>`/`<legend>` combination with a `<label>`, pointing at the text input, to ensure it has an accessible name.
* The range input references the `<label>` with an `aria-labelledby` to get its own accessible name.
* The range input gains an `aria-controls` attribute, pointing at the text input (while acknowledging that `aria-controls` has little support at the time of writing).

## Testing Instructions
* Launch storybook: `npm run storybook:dev`
* Go to the `HeightControl` component
* Check that both fields have accessible names (using the inspector)

### Testing Instructions for Keyboard
Nothing has functionally changed, so both inputs should remain as expected
